pkg/asset/manifests/nutanix/infrastructure.go

```diff
@@ -27,7 +27,7 @@ func GetInfrastructureNutanixPlatformSpec(ic *installconfig.InstallConfig) (*con
 		if err != nil {
 			return nil, fmt.Errorf("unable to connect to Prism Central %s: %w", nutanixPlatform.PrismCentral.Endpoint.Address, err)
 		}
-		pe, err := nc.V3.GetCluster(nutanixPlatform.PrismElements[0].UUID)
+		pe, err := nc.V3.GetCluster(context.TODO(), nutanixPlatform.PrismElements[0].UUID)
```

Comment:
context.TODO should be replaced with an appropriate timeout

Reply:
Fixed.